examples/python/grizzly/birth_analysis_grizzly.py

```diff
@@ -0,0 +1,53 @@
+import pandas as pd
+import grizzly.grizzly as gr
+import numpy as np
+import time
+import sys
+
+passes = ["loop-fusion", "infer-size", "short-circuit-booleans", "predicate", "vectorize", "fix-iterate"]
+years = range(1880, 2011)
+pieces = []
+#columns = ['year', 'name', 'sex', 'births']
+columns = ['name', 'sex', 'births']
+for year in years:
+    path = 'data/names/yob%d.txt' % year
+    frame = pd.read_csv(path, names=columns)
+    frame['year'] = year
+    pieces.append(frame)
+
+# Concatenate everything into a single DataFram
+names = pd.concat(pieces, ignore_index=True)
+print "Size of names: %d" % len(names)
+
+def get_top1000(group):
+    # Note that there is a slight difference that arises
+    # with the name 'Leslyn', year '25' missing as the ordering
+    # in pandas for rows witht he same 'sort' value changes.
+    return group.sort_values(by='births', ascending=False).slice(0, 1000)
+
+#Time preprocessing step
```

Comment:
Nit: Put a space here between `#` and `T`

Reply:
Addressed.